infrastructure/kube/keep-test/monitoring/monitoring-ingress.yaml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
I don't see this file referenced anywhere. Should we add it to `README.md`?

Reply:
2a0f8f385af5dac04897c37ca6377ddeacab1fbb